sync/handler_blocks_request.go

```diff
@@ -24,7 +24,7 @@ func (handler *blocksRequestHandler) ParseMessage(m message.Message, initiator p
 
 	if handler.peerSet.NumberOfOpenSessions() > handler.config.MaxOpenSessions {
 		handler.logger.Warn("we are busy", "message", msg, "pid", initiator)
-		response := message.NewBlocksResponseMessage(message.ResponseCodeBusy,
+		response := message.NewBlocksResponseMessage(message.ResponseCodeRejected, message.ResponseCodeRejected.String(),
```

Comment:
The reason should set to: "busy", or "too many request".
We want to inform the caller, why the request is rejected.

Reply:
message of `message.ResponseCodeRejected.String()` is "too many requests". so I don't think so that we should change this section.